press/api/bench.py

```diff
@@ -563,10 +563,11 @@ def deploy(name, apps_to_ignore=[]):
 	if rg.deploy_in_progress:
 		frappe.throw("A deploy for this bench is already in progress")
 
-	candidate = rg.create_deploy_candidate(apps_to_ignore)
-	candidate.deploy_to_production()
+	candidates = rg.create_deploy_candidate(apps_to_ignore)
```

Comment:
```suggestion
	candidates = rg.create_deploy_candidates(apps_to_ignore)
```

It should be renamed to indicate current state now.